hpedockerplugin/fileutil.py

```diff
@@ -79,6 +79,13 @@ def mkdir_for_mounting(path):
             uuid = data[3]
         else:
             uuid = data[4]
+
+        if mount_prefix:
+            global prefix
```

Comment:
If mount_prefix is supported per backend, then using global variable can be troublesome when parallel mount requests with different backends having different mount_prefix come in. Why don't we just use mount_prefix instead which is safe?